### DISCUSSION TITLE
Win11 22H2 update

### DISCUSSION THREAD

**Discussion by MikeCernea** [FarGroup/FarManager#562] [General]
Hello, after the Windows 11 update from November to 22H2 most applications, far included, starts as terminal tab instead as stand-alone application.
This means 2 things:
 - no longer can I have an icon in taskbar for FAR because all instances are seen as "Windows Terminal"
 - I can no longer configure the font, startup position, etc

Does anyone know a workaround for this inconvenience?

**Comment by yegor-mialyk** [comment]
I do not think there is way to change an  icon in taskbar, because now it's a Windows Terminal one and not Far manager.

As for the window size: Settings -> Startup -> Launch Size

As for the window position:
 - open settings.json
 - add to json (coords are in pixels I think):
```
 "initialPosition": "100,100"
```

As for the font, you can create a Far Manager profile, Settings -> Far Manager -> Additional Settings -> Appearance

**Comment by alabuzhev** [comment]
>  most applications, far included, starts as terminal tab instead as stand-alone application

It can be changed in [settings](https://devblogs.microsoft.com/commandline/windows-terminal-as-your-default-command-line-experience/#default-terminal-setting).
Also, as described in https://github.com/FarGroup/FarManager/issues/551#issuecomment-1272139392, there is a way to run Far (or anything else) without Terminal by prepending `conhost.exe` to the command line (e.g. by modifying the shortcut).

> no longer can I have an icon in taskbar for FAR because all instances are seen as "Windows Terminal"
> I do not think there is way to change an icon in taskbar, because now it's a Windows Terminal one and not Far manager.

`[x] Console icon` in Interface settings. It works in Far 6039 or newer.

> I can no longer configure the font, startup position, etc

I think all that can be changed in Terminal settings, it is quite flexible.
Also, the Terminal developers are open to discussions and ideas. If you think something is missing, [reporting it](https://github.com/microsoft/terminal/issues/new/choose) might help.
